contrib/mesos/pkg/scheduler/scheduler.go

```diff
@@ -449,7 +449,7 @@ func (k *KubernetesScheduler) StatusUpdate(driver bindings.SchedulerDriver, task
 func (k *KubernetesScheduler) reconcileTerminalTask(driver bindings.SchedulerDriver, taskStatus *mesos.TaskStatus) {
 	task, state := k.taskRegistry.UpdateStatus(taskStatus)
 
-	if (state == podtask.StateRunning || state == podtask.StatePending) && taskStatus.SlaveId != nil &&
+	if (state == podtask.StateRunning || state == podtask.StatePending) &&
```

Comment:
I expect that. But it's not specified.

I think we have to try to avoid depending on any detail of those errors as much as possible. They are nice for logging, but otherwise we will depend a lot on the Mesos version.